src/scythe-bidder/create-room.tsx

```diff
@@ -2,31 +2,91 @@
 
 import React from "react";
 import { jsx } from "@emotion/core";
-import { Button, Card, Form, notification, Select, Switch } from "antd";
+import { Button, Card, Form, notification, Select, Modal } from "antd";
 import client from "./client";
-import { FACTIONS_BASE, FACTIONS_IFA, MATS_BASE, MATS_IFA } from "./constants";
+import {
+  FACTIONS_BASE,
+  FACTIONS_IFA,
+  FACTIONS_HI,
+  FACTIONS_LO,
+  MATS_BASE,
+  MATS_IFA,
+  MATS_HI,
+  MATS_LO,
+} from "./constants";
 import { MAX_PLAYERS_BASE, MAX_PLAYERS_IFA, MIN_PLAYERS } from "./constants";
 import { SCYTHE_BIDDER } from "./constants";
+import { QuestionCircleFilled } from "@ant-design/icons";
 
 export default function CreateRoom({ onCreate }: { onCreate: () => void }) {
   const [numPlayers, setNumPlayers] = React.useState(2);
   const [isIfaActive, setIsIfaActive] = React.useState<boolean>(true);
+  const [activeCombinations, setActiveCombinations] = React.useState<string>(
+    "IFA"
+  );
   const maxPlayers = isIfaActive ? MAX_PLAYERS_IFA : MAX_PLAYERS_BASE;
 
+  const { Option } = Select;
+
+  const gameSettingLabel = [
```

Comment:
Does this need to be an array?

Reply:
I tried an object initially but it would not render the QuestionFilledCircle correctly unless it was an array

Reply:
nope, you are right! fixed